Tests/SonarScanner.MSBuild.PreProcessor.Test/JreResolution/LocalJreTruststoreResolverTests.cs

```diff
@@ -40,303 +38,269 @@ public static void Cleanup() =>
         globalEnvScope?.Dispose();
 
     [TestMethod]
-    public void UnixTruststorePath_NullArgs_Throws()
+    public void UnixTruststorePath_NullArgs_Throws() =>
+        FluentActions.Invoking(() => new LocalJreTruststoreResolver(Substitute.For<IProcessRunner>(), new TestRuntime()).UnixTruststorePath(null))
+            .Should().Throw<ArgumentException>()
+            .WithParameterName("args");
+
+    [TestMethod]
+    public void UnixTruststorePath_SystemPathNull_ShouldBeNull()
     {
-        // Arrange
-        var sut = new LocalJreTruststoreResolver(Substitute.For<IFileWrapper>(), Substitute.For<IDirectoryWrapper>(), Substitute.For<IProcessRunner>(), Substitute.For<ILogger>());
+        var runtime = new TestRuntime();
+        var processRunner = Substitute.For<IProcessRunner>();
+        processRunner.Execute(Arg.Any<ProcessRunnerArguments>())
+            .Returns(new ProcessResult(true, string.Empty, string.Empty));
```

Comment:
I don't like random EOLs in short lines
```suggestion
        processRunner.Execute(Arg.Any<ProcessRunnerArguments>()).Returns(new ProcessResult(true, string.Empty, string.Empty));
```